onlinejudge/_implementation/utils.py

```diff
@@ -333,7 +333,7 @@ def snip_large_file_content(content: bytes, limit: int, head: int, tail: int, bo
     except UnicodeDecodeError as e:
         return str(e)
     font = log.bold if bold else (lambda s: s)
-    char_in_line, _ = shutil.get_terminal_size()
+    char_in_line, _ = shutil.get_terminal_size(fallback=(80, 24))  # fallback is required for Circle CI. see https://github.com/kmyk/online-judge-tools/issues/609
```

Comment:
```suggestion
    char_in_line, _ = shutil.get_terminal_size()
    char_in_line = max(char_in_line, 40)
```

- https://github.com/aws/aws-cdk/issues/2253
- https://github.com/aws/aws-cdk/pull/2355
流石に似た事例があるだろうと思い調べてみました、(typescriptですが)ほとんど同じように見えます。
こちらでは単純に40とターミナルサイズのmaxを取っているようです